api/src/pcapi/routes/public/individual_offers/v1/endpoints.py

```diff
@@ -295,6 +299,36 @@ def post_event_offer(
     return serialization.EventOfferResponse.build_event_offer(created_offer)
 
 
+@blueprint.v1_blueprint.route("/events/<int:event_id>/price_categories", methods=["POST"])
+@spectree_serialize(
+    api=blueprint.v1_schema, tags=[EVENT_OFFERS_TAG], response_model=serialization.PriceCategoriesResponse
+)
+@api_key_required
+def post_event_price_categories(
+    event_id: int, body: serialization.PriceCategoriesCreation
+) -> serialization.PriceCategoriesResponse:
+    """
+    Post price categories to be linked to stocks.
```

Comment:
Create price categories for an offer

Reply:
Pour les évènements dans l'api d'offre indiv, on utilise le mot `dates` plutot que `stocks`